src/cliAgent.ts

```diff
@@ -86,9 +86,10 @@ export async function runRestAgent(restConfig: AriesRestConfig) {
     ...afjConfig
   } = restConfig
 
-  const logger = new TsLogger(logLevel ?? LogLevel.error)
+  const logger = PinoLogger.child({ module: 'cloudagent' })
 
-  const agentConfig: InitConfig = {
+  // due to InitConfig interface being tightly coupled to credo-ts type
+  const agentConfig: unknown = {
```

Comment:
not a massive fan of this. We should probably create an adapter so we know the logger we pass to `credo-ts` is compatable with `credo-ts`. Otherwise a missive method could throw some random error.